generator/cli/generate-type-script.ts

```diff
@@ -446,6 +449,11 @@ ${indentBlock(4, escapeBackticks(f.content))}
             ...parameters,
           }
 
+          this.migrationSql = MigrationTransformer.selectSql({
+            template: this._tag,
+            parameters: parameters_
+          })
```

Comment:
I think the code inside the transformer makes sense, but I'm not sure it makes sense as a "transformer". I get that we want to keep the code _inside_ the source code string as small as possible though. Just maybe don't call it a transformer. Maybe the concept is just logic? So something like:

```
src/
  logic/
    index.ts
    migrationSql/
      index.ts
```
```ts
Logic.MigrationSql.select(...)
```

?